RestaurantReservation.Db/Repositories/IRestaurantRepository.cs

```diff
@@ -0,0 +1,12 @@
+﻿using RestaurantReservation.Db.Models;
+
+namespace RestaurantReservation.Db.Repositories
+{
+    public interface IRestaurantRepository
```

Comment:
add documentation (description) for all methods u introduced in your services and repos